test/unit/machine-learning/machine-learning-api-client.spec.ts

```diff
@@ -64,9 +64,14 @@ describe('MachineLearningApiClient', () => {
     },
   };
 
+  const PROJECT_ID = 'test-project';
+  const PROJECT_NUMBER = '1234567';
+  const OPERATION_ID = '987654';
+  const OPERATION_NAME = `projects/${PROJECT_NUMBER}/operations/${OPERATION_ID}`;
```

Comment:
Are we actually using project numbers in our implementation?

Reply:
Yes, the backend returns operations with project number.